README.md

```diff
@@ -230,107 +230,137 @@ The `ping()` function is the only exception to this rule. It does not return hea
+   Then the library will return a _Promise_ for every asynchronous call:
+   ```js
+   mydb.list().then(function(data) {
+     console.log(data);
+   }).catch(function(err) {
+     console.log('something went wrong', err);
+   });
+   ```
 
-Note that all subsequent requests made using this client will also use cookie authentication. The library will automatically refresh the cookie on any `401` or `403` response.
+4. The retry plugins
 
-If you don't specify a username and password during the client construction then cookie authentication is disabled.
+    - `retry429` (or `retry`)
 
-#### The `promises` plugin
+      On occasion Cloudant's multi-tenant offering may reply with an HTTP 429 response because you've exceed the number of API requests in a given amount of time. The `retry429` plugin will automatically retry your request with an exponential back-off.
 
-For example:
+      _For example:_
+      ```js
+      var cloudant = new Cloudant({ url: myurl, maxAttempt: 5, retryInitialDelayMsecs: 1000, plugin: 'retry429' });
+      var mydb = cloudant.db.use('mydb');
+      ```
 
-```js
-var cloudant = new Cloudant({ url: myurl, plugin: 'promises' });
-var mydb = cloudant.db.use('mydb');
-```
+    - `retry5xx`
 
-Then the library will return a _Promise_ for every asynchronous call:
+      The plugin will automatically retry 5xx responses with an exponential back-off.
 
-```js
-mydb.list().then(function(data) {
-  console.log(data);
-}).catch(function(err) {
-  console.log('something went wrong', err);
-});
-```
+      _For example:_
+      ```js
+      var cloudant = new Cloudant({ url: myurl, retryDelayMultiplier: 3, plugin: 'retry5xx' });
+      var mydb = cloudant.db.use('mydb');
+      ```
+
+    - `retryerror`
 
-#### The retry plugins
+      The plugin will automatically retry request errors (e.g. connection reset errors) with an exponential back-off.
 
-1. `retry429`
-2. `retry5xx`
-3. `retryerror`
+      _For example:_
+      ```js
+      var cloudant = new Cloudant({ url: myurl, maxAttempt: 10, plugin: 'retry' });
+      var mydb = cloudant.db.use('mydb');
+      ```
 
-For example, when initialising the Cloudant library you can opt to use the `retry429` plugin:
+#### Using Multiple Plugins
 
+You can pass the plugins as an array, for example:
 ```js
-var cloudant = new Cloudant({ url: myurl, plugin: 'retry429' });
+var cloudant = new Cloudant({ url: myurl, plugins: [ 'cookieauth', 'promises', { retry5xx: { retryDelayMultiplier: 4 } } ] });
 var mydb = cloudant.db.use('mydb');
+mydb.get('mydoc', function(err, data) {
+  console.log(`Document contents: ${data.toString('utf8')}`);
+});
 ```
 
-Then use the Cloudant library normally. You may also opt to configure the retry parameters:
+Note that `plugins` is an alias for `plugin`. Both will accept single or multiple plugin configurations. They are interchangeable.
```

Comment:
Need we bother with this alias given that we've got other breaking changes? I guess you're allowing backwards compatibility for cases like `plugin: 'retry'` but given the configuration is going to move we won't be fully backwards compatible so maybe it is more confusing to provide half of it?

Reply:
Removed legacy `plugin` parameter in 646f7bfffee685e143d53c37a26622229a26b6eb.